robyn/__init__.py

```diff
@@ -30,15 +27,23 @@ def __init__(self, file_object: str) -> None:
         self.file_path = file_object
         self.directory_path = directory_path
         self.config = Config()
+
+        load_vars(project_root=directory_path)
+        logging.basicConfig(level=self.config.log_level)
+
+        # If we are in dev mode, we need to setup the reloader
+        # This process will be used by the watchdog observer while running the actual server as children processes
+        if self.config.dev and not os.environ.get("IS_RELOADER_SETUP", False):
```

Comment:
@AntoineRR , 1) you can initialize the event here and pass is to `setup_reloader`